### ISSUE TITLE
Azure Container Issue in SPFX

### ISSUE THREAD

**Issue by sidjustsid** [SharePoint/sp-dev-fx-webparts#179]
- [* ] Question

#### Expected or Desired Behavior
When we package the SPFX solution to Azure, old references should not be reflected in the new package.

#### Observed Behavior
When we package the SharePoint framework client side web part , files are moved to new container, but when we refresh the page, Old container files also reflecting. After few clicks of refresh new files are reflecting

Any help is much appreciated since we are in the final stage of Integration.

**Comment by mcmynn83** [comment]
Hi There,
I'm just trying to understand the problem better.  Are you saying that you have a js file that you move to a different URL, but when you make the change, upload the package and run the app it still shows your old urls for a while?  

The way that packaging/caching of manifests works is that the browser session that uploads the new package gets a cookie placed on their machine that tells the server not to use our cache.  Everybody else will continue to use the old package values until the cache has updated (which can take up to 20 minutes).

**Comment by sidjustsid** [comment]
Hi @mcmynn83 
Previously we have created a blob and moved our JS files in to that . After some days we have created a new blob and moved our packaging to the new blob.


But when we load the page with new package , we are getting errors saying old container is not found which is referring to  the old blob (Endpoint) .
Because of which our web parts are not loading  until we refresh the page for multiple times.

Would be thank full for the solution since we are moving this to production soon.
